.phpcs.xml

```diff
@@ -0,0 +1,363 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
What is the purpose of this file? 
A proper way to run `phpcs` is `vendor/bin/phpcs --standard=Drupal .`

Reply:
It's an abbreviated Drupal coding standard that I copied from core. It allows you to forgo some of the redundant Documentation of variables. One of my pet peeves about the drupal standard is it requires all this redundant documentation of param and option variables that doesn't really add anything to the understanding of the code. If a param is a class type, it really shouldn't need a comment. The type of PantheonGuzzle is sufficient for the reader of the code to understand what the variable is and what it's used for.

Reply:
Changing the existing Drupal standards check list makes it non-Drupal compliant anymore.